### PR TITLE
[FIX] searchPages in vnda

### PR DESCRIPTION
**Pull request by gabrielMatosBoubee**
In this pr we fix how we verify if is a searchPage or no.

### CODE REVIEW

**Review by guitavano — COMMENTED** [review]
I prefer this prop coming from VNDA app, what do you think?

File: vnda/loaders/productListingPage.ts [hunk 1]
```diff
@@ -101,7 +104,7 @@ const searchLoader = async (
   const sort = url.searchParams.get("sort") as Sort;
   const page = Number(url.searchParams.get("page")) || 1;
 
-  const isSearchPage = url.pathname === "/busca";
+  const isSearchPage = url.pathname === "/busca" || url.pathname === "/s" || url.pathname === props.searchPagePath;
```
Comment (guitavano) [inline vnda/loaders/productListingPage.ts:107]: If url.pathname is undefined, ok verify /busca and /s. But if it is filled, we should look only to thath
```suggestion
  const isSearchPage = props.searchPagePath ? props.searchPagePath ===  url.pathname : url.pathname === "/busca" || url.pathname === "/s"```

**Comment by gabrielMatosBoubee** [comment]
> I prefer this prop coming from VNDA app, what do you think?

You have a point, but I prefer to still this prop here because I think is more UI/UX

**Comment by guitavano** [comment]
Should I say what is my search page in each place that I use this loader? 

> > I prefer this prop coming from VNDA app, what do you think?
> 
> You have a point, but I prefer to still this prop here because I think is more UI/UX

Should I say what is my search page in each place that I use this loader?

**Comment by gabrielMatosBoubee** [comment]
> what is my search page in each place that I use this loader?

I think you need to say it only in the searchPage or some variation. In my mind make more sense select it only when you use

**Comment by guitavano** [comment]
@mcandeia what do you think?

**Review by guitavano — APPROVED** [review]

**Pull request merged**